README.md

```diff
@@ -12,6 +12,7 @@ Signed,
 * [Dylan DPC (@dpc)](https://github.com/dpc)
 * [Erin Power (@XAMPPRocky)](https://github.com/XAMPPRocky)
 * [Jeremy Lempereur (@o0Ignition0o)](https://github.com/o0Ignition0o)
+* [Daniel Egger (@therealprof)](https://github.com/therealprof)
 
```

Comment:
* [Mustafa Karakaplan (@karakaplanm)](https://github.com/karakaplanm)

Reply:
I can't add your signature from here, I added it to the README, is that okay?

Reply:
Yes. It is okay. Thanks.